ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,37 @@
+Make sure you are using the **latest** version of youtube-dl. Latest youtube-dl version at this moment is **2016.03.06**. Read [our FAQ](https://github.com/rg3/youtube-dl/blob/master/README.md#how-do-i-update-youtube-dl) if you have troubles updating.
+- [ ] I've verified that I'm running youtube-dl version **2016.03.06**
+
+*The log (next question) is not required when asking a question or when the purpose of this issue does not imply the need of a log*
```

Comment:
Yes. The main point is emphasize each case when it's required since average user usually has no idea whether he should or not.